Testing/Functional/SmokeTest/SmokeTest002.Tests.ps1

```diff
@@ -0,0 +1,84 @@
+
+#>
+
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSReviewUnusedParameter', 'OrganizationDomain', Justification = 'False positive as rule does not scan child scopes')]
+param (
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string]
+    $OrganizationDomain,
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string]
+    $OrganizationName
+)
+
+Import-Module Selenium
+
+Describe -Tag "UI","Chrome" -Name "Test Report with <Browser> for $OrganizationName" -ForEach @(
+    @{ Browser = "Chrome"; Driver = Start-SeChrome -Arguments @('start-maximized') 2>$null }
+){
+	BeforeAll {
+        $ReportFolders = Get-ChildItem . -directory -Filter "M365BaselineConformance*" | Sort-Object -Property LastWriteTime -Descending
+        $OutputFolder = $ReportFolders[0]
+        $BaselineReports = Join-Path -Path $OutputFolder -ChildPath 'BaselineReports.html'
+        #$script:url = ([System.Uri](Get-Item $BaselineReports).FullName).AbsoluteUri
+        $script:url = (Get-Item $BaselineReports).FullName
+        Open-SeUrl $script:url -Driver $Driver 2>$null
+	}
+
+    Context "Check Main HTML" {
+        BeforeAll {
+            $TenantDataElement = Get-SeElement -Driver $Driver -Wait -ClassName "tenantdata"
+            $TenantDataRows = Get-SeElement -Target $TenantDataElement -By TagName "tr"
+            [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', 'TenantDataColumns',
+                Justification = 'Variable is used in another scope')]
+            $TenantDataColumns = Get-SeElement -Target $TenantDataRows[1] -By TagName "td"        }
+        It "Verify Tenant"{
+
+            $Tenant = $TenantDataColumns[0].Text
+            $Tenant | Should -Be $OrganizationName -Because $Tenant
+        }
+
+        It "Verify  Domain"{
+            $Domain = $TenantDataColumns[1].Text
+            $Domain | Should -Be $OrganizationDomain -Because "Domain is $Domain"
+        }
+    }
+
+    Context "Navigation to detailed reports" {
+        It "Navigate to <Product> (<LinkText>) details" -ForEach @(
+            @{Product = "aad"; LinkText = "Azure Active Directory"}
+            @{Product = "defender"; LinkText = "Microsoft 365 Defender"}
+            @{Product = "onedrive"; LinkText = "OneDrive for Business"}
+            @{Product = "exo"; LinkText = "Exchange Online"}
+            @{Product = "powerplatform"; LinkText = "Microsoft Power Platform"}
+            @{Product = "sharepoint"; LinkText = "SharePoint Online"}
+            @{Product = "teams"; LinkText = "Microsoft Teams"}
+        ){
+            $DetailLink = Get-SeElement -Driver $Driver -Wait -By LinkText $LinkText
+            $DetailLink | Should -Not -BeNullOrEmpty
+            Invoke-SeClick -Element $DetailLink
+
+            Open-SeUrl -Back -Driver $Driver
+        }
+    }
```

Comment:
Can we add a sanity check on the table?  For example, grab all `<table>` and/or `<tr>` tags and ensure the # is greater than 0 for each document and/or each policy statement?

Reply:
Yes.  I think these are value added tests aligned with smoke test purpose.